src/App/ComponentsDocumentation/components/Slab/index.js

```diff
@@ -1,237 +1,76 @@
-
-            <div className="slab slab-secondary">
-                <h4>Here is some text and then a striped item list</h4>
-                <small>
-                    Some text with a small listing of items underneath.
-                </small>
-                <ul className="list item-list">
-                    <li>Item 1</li>
-                    <li>Item 2</li>
-                    <li>Item 3</li>
-                </ul>
-            </div>
-
-            <div className="slab slab-plain">
-                <h4>Here is some text and then a striped item list</h4>
-                <small>
-                    Some text with a small listing of items underneath.
-                </small>
-                <ul className="list item-list">
-                    <li>Item 1</li>
-                    <li>Item 2</li>
-                    <li>Item 3</li>
-                </ul>
-            </div>
-        </ComponentPreview>
-        <p>
-        If no slab extension class is added, it is defaulted to white. In this example we use the utility class <CodeTags type="secondary" code=".border" /> to make it easier for us to see the actual slab container.
-        </p>
-        <ComponentPreview language="html" showCasePanel codeFigure>
-            <div className="slab border">
-                <h4>Here is some text and then a striped item list</h4>
-                <small>
-                    Some text with a small listing of items underneath.
-                </small>
-                <ul className="list item-list">
-                    <li>Item 1</li>
-                    <li>Item 2</li>
-                    <li>Item 3</li>
-                </ul>
-            </div>
-        </ComponentPreview>
-    </>
-);
-
-const SlabBorderTop = () => (
-    <>
-        <h2 id="slab-border-top">Slab with top border</h2>
-        <p>Slab with top border can be used to display, for instance, pros and cons lists.</p>
-        <ComponentPreview language="html" showCasePanel codeFigure>
-            <div className="row">
-                <div className="col-6">
-                    <div className="slab slab-plain slab-border-success">
-                        <h3>Do</h3>
-                        <ul className="list list-bullet">
-                            <li>Do this</li>
-                            <li>Do that</li>
-                            <li>Do the other</li>
-                        </ul>
+const basename = process.env.basename;
+
+const OverviewSlab = () => (
+    <section id="slab-doc">
+        <h2 id="default-slab">Overview</h2>
+        <ComponentPreview language="html" showCasePanel showCasePanelAdvanced={overviewSlabs} codeFigure/>
+        <h3>Things to consider</h3>
+        <ul className="list list-bullet">
+            <li>When using the slab component with a combination of other elements on it, we need to consider how the border-radius of the slab it is affecting the border-radius of the element which takes place as content in the slab.</li>
```

Comment:
this sentence tho.... 
I completely rewrote it.. haha, is this ok? 
"When combining other components with the slab, one must consider how the slab's border-radius affects the edges of the on-top placed component. "